lib/audit_test.pm

```diff
@@ -73,12 +73,10 @@ sub upload_audit_test_logs {
     my ($testcase) = @_;
     upload_logs("$current_file");
 
-    # For syscall test case, different arch has different baseline file,
+    # For some test cases, different arch has different baseline file,
     # so we need to upload the correct file
```

Comment:
Base line file is ARCH specific, so we need upload the correct file the each platform